packages/design-system/src/components/OcTable/OcTable.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <table v-bind="extractTableProps()">
+  <table v-bind="extractTableProps()" class="has-item-context-menu">
```

Comment:
is there a reason this is a method? extractTableProps seems to just return another computed

Reply:
I guess this was implemented to have all the extractTable* stuff next to each other